middleware/log/log_test.go

```diff
@@ -0,0 +1,48 @@
+package log
+
+import (
+	"bytes"
+	golog "log"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+type erroringMiddleware struct{}
+
+func (erroringMiddleware) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error) {
+	return http.StatusNotFound, nil
+}
+
+func TestLoggedStatus(t *testing.T) {
+	var f bytes.Buffer
+	var next erroringMiddleware
+	rule := Rule{
+		PathScope: "/",
+		Format:    DefaultLogFormat,
+		Log:       golog.New(&f, "", 0),
+	}
+
+	log := Logger{
```

Comment:
If this is renamed to `l` or `logger`, then we don't have to alias `log` to `golog` - might make the tests a little more readable. Seeing `log.ServeHTTP` below was a little jarring :smile: